providers/src/airflow/providers/edge/CHANGELOG.rst

```diff
@@ -27,6 +27,22 @@
 Changelog
 ---------
 
+0.9.0pre0
```

Comment:
Aren't we supposed to merge these two ?

Reply:
It is an "internal breaking change" to remove these bindings. So I thought it is valid to get to a 0.9.0... but don't fear this means we are close to 1.0... still I assume there will be a 0.10.0 and 0.11.0 probably :-D